src/components/Login/Login.tsx

```diff
@@ -4,6 +4,12 @@ import { useLocation, useNavigate, useSearchParams } from 'react-router-dom';
 import { Button } from '../../lib/Button';
```

Comment:
looks like the `isSignedIn` and the related `useEffect` in this file are no longer required. Also, can you confirm that all redirects are working as expected whether `basePath` is an empty string or has a value e.g. `fastauth`